xml/System.Security.Cryptography/HashAlgorithmName.xml

```diff
@@ -280,7 +280,11 @@
       <Docs>
         <summary>Gets the underlying string representation of the algorithm name.</summary>
         <value>The string representation of the algorithm name, or <see langword="null" /> or <see cref="F:System.String.Empty" /> if no hash algorithm is available.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+May be `null` or empty to indicate that no hash algorithm is applicable.
```

Comment:
The value property already says this. I don't think we need to say it again.

Reply:
Removed.